pkg/storaged/crypto/keyslots.jsx

```diff
@@ -266,22 +266,23 @@ function ensure_package_installed(steps, progress, package_name) {
     }
 
     progress(cockpit.format(_("Checking for $0 package"), package_name), null);
-    return check_missing_packages([package_name], null)
-            .then(data => {
-                progress(null, null);
-                if (data.missing_names.length + data.unavailable_names.length > 0)
-                    steps.push({
-                        title: cockpit.format(_("The $0 package must be installed."), package_name),
-                        func: progress => {
-                            if (data.remove_names.length > 0)
-                                return Promise.reject(cockpit.format(_("Installing $0 would remove $1."), name, data.remove_names[0]));
-                            else if (data.unavailable_names.length > 0)
-                                return Promise.reject(cockpit.format(_("The $0 package is not available from any repository."), name));
-                            else
-                                return install_missing_packages(data, status_callback(progress));
-                        }
-                    });
-            })
+    return getPackageManager().then(pk => {
+        pk.check_missing_packages([package_name]).then(data => {
+            progress(null, null);
+            if (data.missing_names.length + data.unavailable_names.length > 0)
+                steps.push({
+                    title: cockpit.format(_("The $0 package must be installed."), package_name),
+                    func: progress => {
+                        if (data.remove_names.length > 0)
+                            return Promise.reject(cockpit.format(_("Installing $0 would remove $1."), name, data.remove_names[0]));
+                        else if (data.unavailable_names.length > 0)
+                            return Promise.reject(cockpit.format(_("The $0 package is not available from any repository."), name));
+                        else
+                            return pk.install_missing_packages(data, status_callback(progress));
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22612-2d6fdce8-20251121-125419-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)